scss/components/_state-screen.scss

```diff
@@ -30,6 +30,7 @@
     @extend .oo-expand;
     text-decoration:none;
     cursor: default;
+    z-index: 1000;
```

Comment:
Do we have a place where we define constants for Z-indexes?  If so can we use those.